{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/validation/notebooks/ModelValidation.py

```diff
@@ -131,40 +153,43 @@ def get_targets_from_recipe():
 
 # COMMAND ----------
 
-from model_validation_input import (
-    enable_baseline_comparison,
-    get_prod_workspace_validation_input,
-    get_staging_workspace_validation_input,
-    get_model_type,
-    get_validation_thresholds,
-    get_custom_metrics,
-    get_evaluator_config,
-)
-
 # take input
-enable_baseline_comparison = enable_baseline_comparison()
-
-if env == "prod":
-    data = get_prod_workspace_validation_input(spark)
-elif env == "staging":
-    data = get_staging_workspace_validation_input(spark)
-else:
-    raise Exception(
-        "Unknown environment. Please select 'prod' or 'staging' as environment name"
-    )
-
-targets = get_targets_from_recipe()
-
-model_type = get_model_type_from_recipe()
-
-custom_metrics = get_custom_metrics()
-
-validation_thresholds = get_validation_thresholds()
-
-evaluator_config = get_evaluator_config()
-
-assert data, "Please provide correct data input for " + env
-assert model_type, "Please provide correct model type"
+run_mode = dbutils.widgets.get("run_mode").lower()
+dbutils.widgets.dropdown("enable_baseline_comparison", "false", ["true", "false"], "Enable Baseline Comparison")
+dbutils.widgets.text("validation_input", "", "Validation Input")
+dbutils.widgets.text("model_type", "", "Model Type")
+dbutils.widgets.text("targets", "", "Targets")
+dbutils.widgets.text("custom_metrics_loader_function", "", "Custom Metrics Loader Function")
+dbutils.widgets.text("validation_thresholds_loader_function", "", "Validation Thresholds Loader Function")
+
+enable_baseline_comparison = dbutils.widgets.get("enable_baseline_comparison")
+assert enable_baseline_comparison == "true" or enable_baseline_comparison == "false"
+enable_baseline_comparison = enable_baseline_comparison == "true"
+
+validation_input = dbutils.widgets.get("validation_input")
+assert validation_input
+data = spark.sql(validation_input)
+
+{% if cookiecutter.include_feature_store == "yes" %}model_type = dbutils.widgets.get("model_type")
+targets = dbutils.widgets.get("targets")
+{% else %}model_type = get_model_type_from_recipe()
+targets = get_targets_from_recipe(){% endif %}
+assert model_type
+assert targets
+
+custom_metrics_loader_function_name = dbutils.widgets.get("custom_metrics_loader_function")
+validation_thresholds_loader_function_name = dbutils.widgets.get("validation_thresholds_loader_function")
+assert custom_metrics_loader_function_name
+assert validation_thresholds_loader_function_name
+custom_metrics_loader_function = getattr(
+    importlib.import_module("validation"), custom_metrics_loader_function_name
+)
+validation_thresholds_loader_function = getattr(
+    importlib.import_module("validation"), validation_thresholds_loader_function_name
+)
+custom_metrics = custom_metrics_loader_function()
+validation_thresholds = validation_thresholds_loader_function()
+evaluator_config = {}
```

Comment:
Will this get updated later? We're not using get_evaluator_config()

Reply:
Yes! I'll update before merging. I shouldn't have left it out.